### PR TITLE
Update for avatar upload & fix breaking change on noise-js

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

**Review by WeiJun0827 — APPROVED** [review]

**Review by williamchong — APPROVED** [review]

File: src/util/fileupload.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export async function handleAvatarUploadAndGetURL(user, file, avatarSHA256) {
     filename: `likecoin_store_user_${user}_${IS_TESTNET ? 'test' : 'main'}`,
     mimetype: file.mimetype,
   });
-  return avatarUrl;
+  return `${avatarUrl}&${hash256.substring(0, 7)}`;
```
Comment (williamchong) [inline src/util/fileupload.js]: I think md5 is better in this case, since you only take 7 char substring and md5 is shorter than sha256

**Review by williamchong — APPROVED** [review]
from which lib this `md5-hex` is from though?

**Comment by nwingt** [comment]
> from which lib this `md5-hex` is from though?

@williamchong  ava

**Comment by williamchong** [comment]
`ava` is in `devDependencies`, that means we do need to add `md5-hex` to `dependencies`

**Pull request merged**